packages/wrangler/e2e/pages-dev.test.ts

```diff
@@ -227,4 +574,137 @@ describe("pages dev tests", () => {
 			"Unexpected error building Functions directory"
 		);
 	});
+
+	it("should modify _routes.json during dev session (_worker)", async () => {
```

Comment:
Is this message correct? 
Shouldn't it be "should support modifying ..."?

Reply:
yes indeed. fixed! ✅